package.json

```diff
@@ -24,7 +26,9 @@
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test --env=jsdom",
```

Comment:
i suggest a script for linting : `"lint": "standard --fix"` @Rexben001 @shaunorpen @vunderkind